Constants.java

```diff
@@ -0,0 +1,36 @@
+package assignment_java;
+
+//Constant class to manage hardcoded values
+ public class Constants {
+public static final String BINARY = "binary";
+public static final String OCTAL = "octal";
+public static final String DECIMAL = "decimal";
+public static final String HEXADECIMAL = "hexadecimal";
+public static final String ADDITION = "addition";
+public static final String SUBTRACTION = "subtraction";
+public static final String MULTIPLICATION = "multiplication";
+public static final String INPUT = "Do you want to input more number? (enter 'q' to perform operation or 'y' to input more number): ";

```

Comment:
Use more friendly and readable terms for option such as Done or execute instead of q and y